src/index.ts

```diff
@@ -75,7 +75,11 @@ function handleLeaveRoom(roomid: string, userid: string) {
 
         // Update users in current room or delete room if empty.
         let invitedUserIds = Object.keys(room.invited);
```

Comment:
Is `invitedUserIds` still an accurate name? If it is, should we create another property for users? For example, users can be `invited` or `joined` right? And we want to delete the room if:
1. Only one joined user with 0 invited users
2. No joined users with any number of invited users

Reply:
Right now this is how it's work:

1. All invited users get added to `room.invited`.
2. If a user rejects the invite, they are still in that array because we don't want other users to know that they rejected the file transfer.
3. If a user leaves the room after accepting, they're removed from invited.

Is there any big reason to switch out the logic?

Reply:
why don't we want the sender to know that their file was rejected in 2? Don't we want the sender to know that the file didn't get through?

Reply:
They're rejecting to join the room not necessarily a specific file. Nonetheless I just thought that rejecting creates a "negative" interaction but I guess it could be helpful to know if someone rejected your invite instead of constantly waiting.

Reply:
Ya i think its fine for now, but might be confusing for people if they just stay in the room